pkg/es/config/config.go

```diff
@@ -136,6 +136,8 @@ type Configuration struct {
 	Tags                     TagsAsFields  `mapstructure:"tags_as_fields"`
 	// Enabled, if set to true, enables the namespace for storage pointed to by this configuration.
 	Enabled bool `mapstructure:"-"`
```

Comment:
is this still used? Let's add some comments, because this is not the first time I have this question, and the flag doesn't really make sense to me.